src/Haskell/Ide/Engine/Transport/LspStdio.hs

```diff
@@ -966,6 +966,7 @@ hieOptions :: [T.Text] -> Core.Options
 hieOptions commandIds =
   def { Core.textDocumentSync       = Just syncOptions
       , Core.completionProvider     = Just (J.CompletionOptions (Just True) (Just ["."]))
+      , Core.typeDefinitionProvider = Just (J.GotoOptionsStatic True)
```

Comment:
I looked the `haskell-lsp` docs and was none the wiser afterwards.

Reply:
I know that the protocol allows to pass a link instead of a file as response, which would be nice for hoogle packages. Should we look into this?

Reply:
What would that entail?
What do you mean with hoogle packages? Documentation?

Reply:
I’ll investigate a bit more, I think it would entail showing the local Html page build with haddock or potentially a link to the haddock page for the type, which we can get from hoogle

Reply:
This would also be interesting for `findDefinition`.

Reply:
Also, if it worked project wide, but I think, this is a different problem.

Reply:
I would expect the typedefinitionprovider flag to be set only if there is a handler for the gototypedefinition request, rather than having to be set explicitly. 

So there may be a shortcoming in `haskell-lsp`.